cmd/kinflate/demo/springboot.md

```diff
@@ -0,0 +1,328 @@
+# Demo: SpringBoot
+
+This example takes k8s resources for one Spring Boot application sbdemo, and customizes them to suit a
+production scenario.
```

Comment:
May be can re-word this:
In this tutorial, you will learn - how to use `kinflate` to customize a basic Sprint Boot application's k8s configuration for dev and production use cases.